bin/common.sh

```diff
@@ -92,6 +92,36 @@ if [ "$SAS_COMMON_SOURCED" = "" ]; then
        fi
     fi
 
+    ## Check for air gap deployment
+    AIRGAP_DEPLOYMENT=${AIRGAP_DEPLOYMENT:-false}
+    if [ $AIRGAP_DEPLOYMENT == true ]; then
+      ## Check for AIRGAP_REGISTRY, if null/empty, error out.  Otherwise set and create HELM_URL_BASE.
+      if [ -z $AIRGAP_REGISTRY ]; then
+        log_error "AIRGAP_REGISTRY has not been set"
+        log_error "Please provide the URL for the private image registry and try again"
+        exit 1
+      else
+        AIRGAP_IMAGE_PULL_SECRET_NAME=${AIRGAP_IMAGE_PULL_SECRET_NAME:-"v4m-image-pull-secret"}
+        if [ -z "$AIRGAP_HELM_REPO"]; then
+          log_debug "Separate AIRGAP_HELM_REPO value not provided"
+          log_debug "Setting AIRGAP_HELM_REPO to oci://${AIRGAP_REGISTRY}/"
+          AIRGAP_HELM_REPO="oci://${AIRGAP_REGISTRY}/"
+        fi  
+      fi
+    fi
+
+    ## The user will need to create the namespace and secret before running the deployment scripts.
+    ## This function will produce an error if the secret is not found in the environment.
+    function checkForAirgapSecretToNamespace {
```

Comment:
The function was supposed to be called checkForAirgapSecretInNamespace ... I think I was in between naming ideas and didn't fully change it.  It will be updated in the next push.

I've also made sure to only call it after the namespace has been made so it has to exist.
Cluster Metric Monitoring - It's positioned after the kubectl create $MON_NS line.
Viya Metric Monitoring - The function's only called after the check to make sure $VIYA_NS exists happens so it will have to exist.
Log Monitoring - Assuming the users are exclusively using deploy_logging.sh, it will be after the kubectl create $LOG_NS.  The only way I can think of a namespace not existing is if the customer tries to run one of the independent scripts on the logging side prior to running the deploy_logging.sh (to create the namespace); in that case, it would throw a kubectl error saying the namespace doesn't exist ... but also confirms that the secret wouldn't be in the namespace because it doesn't exist.  I think that's okay.